server-common/src/main/java/org/apache/kafka/server/config/QuotaConfig.java

```diff
@@ -54,9 +54,8 @@ public class QuotaConfig {
 
     public static final String CLIENT_QUOTA_CALLBACK_CLASS_CONFIG = "client.quota.callback.class";
     public static final String CLIENT_QUOTA_CALLBACK_CLASS_DOC = "The fully qualified name of a class that implements the ClientQuotaCallback interface, " +
-            "which is used to determine quota limits applied to client requests. By default, the &lt;user&gt; and &lt;client-id&gt; " +
-            "quotas that are stored in ZooKeeper are applied. For any given request, the most specific quota that matches the user principal " +
```

Comment:
The explanation of the default behavior is still useful (without mentioning zk specifically).

Reply:
I will open a PR to add it back.